### PR TITLE
Updated intro notebook, NarrativeText to Text

### PR DESCRIPTION
**Pull request by ajjimeno**

### CODE REVIEW

File: examples/training/1-Intro to Bricks.ipynb [hunk 1]
```diff
@@ -359,7 +359,7 @@
     {
      "data": {
       "text/plain": [
-       "'[1]\\xa0Geolocated combat footage has confirmed Russian gains in the Dvorichne area northwest of Svatove.'"
+       "'Search form'"
```
Comment (MthwRobinson) [inline examples/training/1-Intro to Bricks.ipynb:362]: Those look like the banners at the top of the page. Are the writeup portions not getting flagged as `NarrativeText` now?

image

Comment (ajjimeno) [reply examples/training/1-Intro to Bricks.ipynb:362]: No NarrativeText is found among the elements.

Comment (ajjimeno) [reply examples/training/1-Intro to Bricks.ipynb:362]: image

**Comment by MthwRobinson** [comment]
@ajjimeno - Closing this PR, #309 fixes the issue.

**Pull request closed without merging**